fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java

```diff
@@ -60,6 +71,7 @@ public class IcebergApiConverter {
     private static final Logger LOG = LogManager.getLogger(IcebergApiConverter.class);
     public static final String PARTITION_NULL_VALUE = "null";
     private static final int FAKE_FIELD_ID = -1;
+    private static final String ALLOW_INCOMPATIBLE_CHANGES = "allow_incompatible_changes";
```

Comment:
what's this? why use this properties?

Reply:
This option allows incompatible changes to be made to a schema, For example, adding a required column and attempting to read old data files without that column will cause a failure in iceberg table.
We can set this option in alter column statement like:
```
alter table ice_table02 
add column col10 INT not null default '0'
PROPERTIES
("allow_incompatible_changes"="true")
```